painttiming.bs

```diff
@@ -64,7 +64,7 @@ This is formally defined as the when <a>update the rendering</a> happens in even
 
 <dfn export>First Paint</dfn> entry contains a {{DOMHighResTimeStamp}} reporting the time when the user agent first rendered after navigation. This excludes the default background paint, but includes non-default background paint and the enclosing box of an iframe. This is the first key moment developers care about in page load – when the user agent has started to render the page.
 
-<dfn export>First Contentful Paint</dfn> entry contains a {{DOMHighResTimeStamp}} reporting the time when the user agent first rendered any text, image (including background images), non-white canvas or SVG. This excludes any content of iframes, but includes text with pending webfonts. This is the first time users could start consuming page content.
+<dfn export>First Contentful Paint</dfn> entry contains a {{DOMHighResTimeStamp}} reporting the time when the user agent first rendered any text, image (including background images and mask images), non-white canvas or SVG. This excludes any content of iframes, but includes text with pending webfonts. This is the first time users could start consuming page content.
```

Comment:
This definition is still too vague. We need to refer to https://drafts.fxtf.org/css-masking-1/#the-mask-image and specify under which condition an image is considered as rendered. e.g. [invalid image](https://drafts.csswg.org/css-images-4/#invalid-image) as defined in https://drafts.csswg.org/css-images-4/#interpolating-images presumably doesn't count. We need to define what happens to a [loading image](https://drafts.csswg.org/css-images-4/#loading-image), and all other scenarios.

Reply:
I tried to address this and other issues in https://github.com/w3c/paint-timing/pull/66